submissions/migrations/0003_submission_null_FK.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='student_item',
+            field=models.ForeignKey(to='submissions.StudentItem', null=True, default=1),
+            preserve_default=False
+        ),
+    ]
```

Comment:
I think so. The two longest lines seem to be these, what else should we be considering when extrapolating?
```
DEBUG:django.db.backends:(0.114) ALTER TABLE `submissions_submission` MODIFY `student_item_id` integer NULL; args=[]
DEBUG:django.db.backends:(0.130) ALTER TABLE `submissions_submission` ADD CONSTRAINT `su_student_item_id_d3801ff833d05b1_fk_submissions_studentitem_id` FOREIGN KEY (`student_item_id`) REFERENCES `submissions_studentitem` (`id`); args=[]
```

There's also now a possibility of avoiding migrations entirely if the approach in https://github.com/edx/edx-submissions/pull/35 wins out as our best option.